src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -565,7 +553,7 @@ public void LoadUrl(string url)
         /// Returns the URL of the Web
         /// <param name="data">The data of Web</param>
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public void LoadHTMLString(string data)
```

Comment:
Use PascalCase for acronyms that are longer than two letters.
```suggestion
        public void LoadHtmlString(string data)
```

Reply:
Done